userguide/sharing.rst

```diff
@@ -203,16 +203,31 @@ information given when the share was created.
    |                              |               |          |                                                                                                               |
    +------------------------------+---------------+----------+---------------------------------------------------------------------------------------------------------------+
    | Hosts Allow                  | string        | ✓        | Enter a list of allowed hostnames or IP addresses. Separate entries with a comma, space, or tab.              |
-   |                              |               |          |                                                                                                               |
+   |                              |               |          | Please see the :ref:`note <hosts allow/deny note>` for more information.                                      |
    +------------------------------+---------------+----------+---------------------------------------------------------------------------------------------------------------+
    | Hosts Deny                   | string        | ✓        | Enter a list of denied hostnames or IP addresses. Separate entries with a comma, space, or tab.               |
-   |                              |               |          |                                                                                                               |
+   |                              |               |          | Please see the :ref:`note <hosts allow/deny note>` for more information.                                      |
    +------------------------------+---------------+----------+---------------------------------------------------------------------------------------------------------------+
    | Auxiliary Parameters         | string        | ✓        | Enter any additional `afp.conf <https://www.freebsd.org/cgi/man.cgi?query=afp.conf>`__ parameters             |
    |                              |               |          | not covered by other option fields.                                                                           |
    |                              |               |          |                                                                                                               |
    +------------------------------+---------------+----------+---------------------------------------------------------------------------------------------------------------+
 
+.. note::
+   :name: hosts allow/deny note
+
+   If neither *Hosts Allow* or *Hosts Deny* contains an entry, then allow
+   access.
```

Comment:
This reads like it's missing objects? How about something like ... "then AFP share access is allowed for any host."   ?